### PR TITLE
[ RCF-988 ] adding Selected handel releated changes to RegistrationPacket class

### PR DESCRIPTION
**Pull request by Khuddusshariff0022**

### CODE REVIEW

File: commons-packet/commons-packet-manager/src/test/java/io/mosip/commons/packet/test/dto/packet/RegistrationPacketTest.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+public class RegistrationPacketTest {
+    @Autowired
+    private RegistrationPacket registrationPacket;
+    @Before
+    public void befor(){
+        registrationPacket =new RegistrationPacket("yyyy-MM-dd'T'HH:mm:ss.SSS");
+    }
+
+    @Test
+    public void setFilesWithArrayOfStringTest()
```
Comment (ase-101) [inline commons-packet/commons-packet-manager/src/test/java/io/mosip/commons/packet/test/dto/packet/RegistrationPacketTest.java:23]: Test case naming convention should be <methodToTest>_<TestCaseName>_<TestResult>
Eg: setField_withArrayOfString_thenPass

File: commons-packet/commons-packet-manager/src/test/java/io/mosip/commons/packet/test/dto/packet/RegistrationPacketTest.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class RegistrationPacketTest {
+    @Autowired
```
Comment (ase-101) [inline commons-packet/commons-packet-manager/src/test/java/io/mosip/commons/packet/test/dto/packet/RegistrationPacketTest.java:15]: give a new line gap between 2 declaration

File: commons-packet/commons-packet-manager/src/test/java/io/mosip/commons/packet/test/dto/packet/RegistrationPacketTest.java (hunk 1 above)
Comment (ase-101) [reply commons-packet/commons-packet-manager/src/test/java/io/mosip/commons/packet/test/dto/packet/RegistrationPacketTest.java:23]: Kindly change the method names of all the test cases defined in this RegistrationPacketTest

**Comment by ase-101** [comment]
Also I don't see pom version changes in the PR @Khuddusshariff0022

File: commons-packet/commons-packet-manager/src/test/java/io/mosip/commons/packet/test/dto/packet/RegistrationPacketTest.java (hunk 1 above)
Comment (Khuddusshariff0022) [reply commons-packet/commons-packet-manager/src/test/java/io/mosip/commons/packet/test/dto/packet/RegistrationPacketTest.java:23]: done

File: commons-packet/commons-packet-manager/src/test/java/io/mosip/commons/packet/test/dto/packet/RegistrationPacketTest.java (hunk 2 above)
Comment (Khuddusshariff0022) [reply commons-packet/commons-packet-manager/src/test/java/io/mosip/commons/packet/test/dto/packet/RegistrationPacketTest.java:15]: done

**Review by ase-101 — APPROVED** [review]

**Pull request merged**